src/Jira/Api.php

```diff
@@ -157,14 +157,14 @@ public function getAttachment($attachmentId)
         return $result;
     }
 
-    public function getProjects()
+    public function getProjects($expand = '')
```

Comment:
I would rather make `$expand` parameter and array and then concatenate values by comma before making API call (right in this method).